src/bundle/Resources/config/services.yaml

```diff
@@ -1,25 +1,28 @@
 imports:
-  - { resource: forms.yaml }
-  - { resource: views.yaml }
+    - { resource: forms.yaml }
+    - { resource: views.yaml }
 
 services:
-  _defaults:
-      autoconfigure: true
-      autowire: true
-      public: false
+    _defaults:
+        autoconfigure: true
+        autowire: true
+        public: false
 
-  Ibexa\Platform\Bundle\Search\Controller\SearchController:
-      tags:
-          - controller.service_arguments
+    Ibexa\Platform\Bundle\Search\Controller\SearchController:
+        tags:
+            - controller.service_arguments
 
-  Ibexa\Platform\Search\Mapper\PagerSearchContentToDataMapper:
-      arguments:
-          $contentTypeService: '@ezpublish.api.service.content_type'
-          $userService: '@ezpublish.api.service.user'
-          $userLanguagePreferenceProvider: '@eZ\Publish\Core\MVC\Symfony\Locale\UserLanguagePreferenceProvider'
-          $translationHelper: '@ezpublish.translation_helper'
-          $languageService: '@ezpublish.api.service.language'
 
-  Ibexa\Platform\Search\QueryType\SearchQueryType:
-      arguments:
-          $searchService: '@ezpublish.api.service.search'
+    Ibexa\Platform\Search\Mapper\PagerSearchContentToDataMapper:
+        arguments:
+            $contentTypeService: '@ezpublish.api.service.content_type'
+            $userService: '@ezpublish.api.service.user'
+            $userLanguagePreferenceProvider: '@eZ\Publish\Core\MVC\Symfony\Locale\UserLanguagePreferenceProvider'
+            $translationHelper: '@ezpublish.translation_helper'
+            $languageService: '@ezpublish.api.service.language'
+
+    Ibexa\Platform\Search\Mapper\PagerDataMapperInterface: '@Ibexa\Platform\Search\Mapper\PagerSearchContentToDataMapper'
```

Comment:
Could we not tie those two together? As I understand those are for autowiring but it should work without it as concrete implementation is specified in service definition.

Reply:
No, it will lead to this error:
```
Cannot autowire service "App\Controller\AlertController": argument "$pagerDataMapper" of method "__construct()" references interface "Ibexa\Platform\Search\Mapper\PagerDataMapperInterface" but no such service exists. You should maybe alias this interface to one of these existing services: "EzSystems\EzPlatformAdminUi\Search\PagerSearchContentToDataMapper", "Ibexa\Platform\Search\Mapper\PagerSearchContentToDataMapper".
```
I will open one more PR to admin-ui bundle to include:
`Ibexa\Platform\Search\Mapper\PagerDataMapperInterface: '@EzSystems\EzPlatformAdminUi\Search\PagerSearchContentToDataMapper'`
Ok?

Reply:
Then that code
`    Ibexa\Platform\Search\Mapper\PagerDataMapperInterface: '@Ibexa\Platform\Search\Mapper\PagerSearchContentToDataMapper'
`

should either go to your App (if you use autowire), or you should inject specifying instance and not relay on it.